src/Core/Domain/CatalogPriceRule/Command/AddCatalogPriceRuleCommand.php

```diff
@@ -191,17 +191,17 @@ public function getPrice()
     /**
      * @return string
      */
-    public function getFrom()
+    public function getDateTimeFrom()
```

Comment:
Why calling this method DateTimeFrom and not DateFrom, it's not related to a DateTime object which is disturbing.
Maybe a better call `fromDate`, `toDate` or if you want to keep the DateTime name, return a real DateTime which is also a good way to play with date.